charts/all-in-one/templates/rudolf-service.yaml

```diff
@@ -0,0 +1,67 @@
+{{ if .Values.rudolfService.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .Values.rudolfService.serviceAccount.roleArn }}
+  labels:
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+  name: {{ $.Chart.Name }}-9c-rudolf-iam-role
+  namespace: {{ $.Chart.Name }}
+
+---
+
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: 9c-rudolf
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+  name: 9c-rudolf
+  namespace: {{ $.Chart.Name }}
+spec:
+  podManagementPolicy: OrderedReady
+  replicas: 1
+  selector:
+    matchLabels:
+      app: 9c-rudolf
+  serviceName: 9c-rudolf
+  template:
+    metadata:
+      labels:
+        app: 9c-rudolf
+    spec:
+      containers:
+        - name: 9c-rudolf
+          env:
+            - name: DATABASE_URL
+              valueFrom: 
+                secretKeyRef:
+                  key: database-url
+                  name: 9c-rudolf-env
```

Comment:
is it from external secret?

Reply:
Yes.

Reply:
then you should also make externalSecret resource like below
https://github.com/planetarium/9c-infra/blob/main/charts/all-in-one/templates/secret-market-db.yaml

Reply:
I applied it in [3987800](https://github.com/planetarium/9c-infra/pull/787/commits/3987800620e38ac3e8db4394374db08a6ea2934f) commit.